java/core/src/test/java/com/microsoft/bond/testmodel/schema.bond

```diff
@@ -7,6 +7,8 @@ struct C;
 struct D;
 struct E<U>;
 
+struct NotHere;
```

Comment:
Not sure what this is about, TBD.

---
In reply to: [125347070](https://github.com/Microsoft/bond/pull/517#discussion_r125347070) [](ancestors = 125347070)